binary-search/test_binary_search.py

```diff
@@ -0,0 +1,15 @@
+import binary_search as binary_search
+import pytest as pytest
```

Comment:
You only need to use `as` is you going to rename the module for use in your code. Since you are assigning the same name as what the module already has, you only need `import`. This code is equivalent to
```python
import binary_search
import pytest
```

Reply:
Also, you never use `pytest` in your tests, so you do not need to import it. You do not need to import `pytest` to be able to run the tests with `pytest`, you only need it when testing for exceptions, etc.